src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -27,6 +28,11 @@ public override void Write(string message)
             {
                 _writer.Write(message);
             }
+
+            public override void WriteLine(string message)
+            {
+                _writer.WriteLine(message);
+            }
```

Comment:
So we have the same behavior but the difference is that in desktop we have a private `Write` method which does the same as the public `Write` method in Core. So they both end up calling `Write` from `DefaultTraceListener.Fail(string)` but in desktop we call the private method which can't be overridden.